README.md

```diff
@@ -1,3 +1,111 @@
-# Node.js Express Web Service using standard `node` image
+# Primer: Node.js Application Hardening
 
-TO-DO
+## The Problem and the Solution 
+
+Creating high-quality container images is hard. How to choose an optimal base image? How to configure a multistage build to make the end slim? How to keep the vulnerability scanners calm? One must be a true container expert to get it right. However, there might be an alternative way.
```

Comment:
`make the end slim` -> `make the end image slim`

P.S. One sentense - one line 😉

Reply:
Yes, thanks :))